### PR TITLE
Only look at js files named app.js or in the lib folder

### PR DESCRIPTION
**Pull request by adammw**
:v:

/cc @zendesk/vegemite

### Description
Only look at js files named app.js or in the lib folder.

### Risks
* None

### CODE REVIEW

**Comment by danielbreves** [comment]
:+1:

File: lib/zendesk_apps_support/package.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def files
     end
 
     def js_files
-      @js_files ||= files.select { |f| f =~ /\.js$/ }
+      @js_files ||= files.select { |f| f =~ /^(lib\/.+|app)\.js$/ }
```
Comment (danielbreves) [inline lib/zendesk_apps_support/package.rb]: Will this work with folders inside lib?

**Comment by timbertson** [comment]
Pet peeve: I know it's longer, but I'd find this much more readable and simpler than a regexp (especially since it maps to how you yourself described the logic):

    file == "app.js" || ( file.start_with?("lib/") && file.end_with?(".js") )

:+1: either way though

**Comment by adammw** [comment]
:haircut:

**Comment by danielbreves** [comment]
:+1:

**Comment by timbertson** [comment]
Yaaay :+1:

**Comment by sandlerr** [comment]
`lib_files` could be just js_files without app.js then

**Comment by ocke** [comment]
or `js_files` is just lib_files with app.js

**Pull request merged**